adv_c.py

```diff
@@ -0,0 +1,129 @@
+     Now your brothers and all the adaptive family want to get together and go talk to the researchers.
+     There was no option for you to avoid meeting them."""
+    game_over(reason)
+
+@when("yes") 
+def triplets():
+    current_room = RoomTriplets
+    print(current_room)
+    say("""A couple years after, you got the worst news of your life, one of the triplets has commited suicide.\n 
+    Who do you think it was?\n
+    david, edward or robert""")
+    
+    @when("david")
+    def david():
+        #the player made wrong choice, call game_over() function with the "reason"
+        reason = """Sorry brave warrior, your guess is wrong. David was adapted by working class family but with loving family he had developed the best way to cope with his mental illness out of the triplets. Since Rober left the restaurant and even more after Edwards death David and Robert drifted apart\n"""
+        game_over(reason)
+    
+    @when("edward")
+        # the player won the game by guessing correct
+    def edward():
+        reason = """Yes, it was Edward. Edward was adapted by the middle class family but he and his father was just not good match. 
+        Edward sturuggle most growing up without necessary support. 
+        He was exhibited increasing sigh of bipoar disorder This research was most cruel to him \n
+        Bravo!! You made the correct guess and won the game\n"""
+        game_over(reason)
+    
+    @when ("robert") 
+    def robert():
+     reason = """Sorry brave warrior, you did not win this time. 
+     Rober was adapted by upper-middle class family whose father was doctor. 
+     They has money but not quority time for him. , 
+     but he has established carrie as a lawyer.\n"""
+     game_over(reason)
+
+def game_over(reason):
+    #say("""Sorry brave warrior, your guess is wrong. David was adapted by working class family but with loving family he had developed the best way to cope with his mental illness out of the triplets.print Since Rober left the restaurant and even more after Edwards death David and Robert drifted apart""")
+    say(reason)
+    print("This game was based on the documetary field \'Three Identical Strangers\' \n \"Three Identical Strangers\". wikipedia, https://en.wikipedia.org/wiki/Three_Identical_Strangers. Accessed 17 Septempber 2021.")
+    print("Study was never publised and all documents were placed with Yale University and restricted until 2065. \nThere were at least six sets of twin/ triplet who got separated in this study and at lease three of the separated sibling apparently committed suicide.")
+    print("Type 'exit' to exit the game, Thank you!!!")
+
+@when("exit")
+def exit():
+ say("exiting the game")
+ keyboard = Controller()
+ keyboard.press(Key.ctrl)
+ keyboard.press('z')
+ keyboard.release(Key.ctrl)
+ keyboard.release('z')
+ keyboard.press(Key.enter) 
+
+start()
```

Comment:
this start function needs to be in a function or else when we import this file, it starts your adventure, instead of waiting for the user to choose that adventure